packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js

```diff
@@ -116,3 +122,67 @@ describe('The user', () => {
     expect(response.headers.location).toBe(IDENTIFY.uri)
   })
 })
+
+describe('takeover by agreed handler once agreed flag is set', () => {
+  const sessionCookieName = 'garibaldi'
+  const getMockRequest = () => ({
+    path: 'crazy-paving',
+    state: {
+      [sessionCookieName]: {
+        id: 'a disappointing currant embedded biscuit'
+      }
+    },
+    cache: () => ({
+      helpers: {
+        status: {
+          get: async () => ({
+            agreed: true
+          })
+        }
+      }
+    })
+  })
+  const getMockToolkit = (takeover = () => {}) => ({
+    state: () => {},
+    redirect: jest.fn(() => ({
+      takeover
+    }))
+  })
+  const initialisedSessionManager = sessionManager(sessionCookieName)
+
+  beforeEach(jest.clearAllMocks)
+
+  it("redirects to agreed handler when status is already agreed and request path isn't in exempt set", async () => {
```

Comment:
```suggestion
  it('redirects to agreed handler when status is already agreed and request path isn't in exempt set', async () => {
```

I know I'm nitpicking here but :)

Reply:
I would normally use single quotes, but note the apostrophe in isn't

Reply:
Oops. You're totally right!